.github/workflows/build-macOS.yml

```diff
@@ -0,0 +1,39 @@
+name: build on macOS
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        runner: [macos-11, macos-12]
+    runs-on: ${{ matrix.runner }}
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: install dependencies
+      run: brew install automake coreutils ffmpeg wxwidgets
+    - name : symlink wxwin.m4
+      run: find /usr/local/Cellar/wxwidgets/ -name "wxwin.m4" -exec ln -s {} /usr/local/share/aclocal/wxwin.m4 \;
+    - name: autogen
```

Comment:
you can actually run all these lines together, like this https://github.com/alexkay/spek/pull/268/files#diff-ec943e7e17a4e6a51aa0dd98515f3c31fabb7172a37eba9ccb06cb124768f051R66-R77

Reply:
Thank you for help.